src/pages/api/auth/[...nextauth].ts

```diff
@@ -80,11 +73,8 @@ export const authOptions: NextAuthOptions = {
     strategy: 'jwt',
     maxAge: 24 * 60 * 60,
   },
-
-  // Used to decorate the home page design
-  theme: {
-    colorScheme: 'dark',
-    logo: '/fintech_logo_final-removebg_white.png',
+  pages: {
+    signIn: '/login',
```

Comment:
I think you can integrate in the signout page here as per the [docs](https://next-auth.js.org/configuration/options#pages)

Reply:
We don't have a custom sign out page so it would be unnecessary to add that now. Plus signing out can just simply be using the NextAuth signOut hook.